### DISCUSSION TITLE
Google api

### DISCUSSION THREAD

**Discussion by thecyberneh** [hahwul/dalfox#338] [Q&A]
What is Found dalfox Google api?

**Comment by hahwul** [comment]
Hi @thecyberneh 
Dalfox basically also has passive detection based on regular expressions. 
I think A value suitable for the Google api pattern was detected and appeared. 

https://github.com/hahwul/dalfox/blob/97c9a57e8c52297eceb7bdde09e49259b1dea6e5/pkg/scanning/grep.go#L41

Since it is marked as [G] in PoC, it can be distinguished from XSS. 
If you don't want to see it, you can exclude it in the following way.

with `--skip-grepping` flag
```
$ dalfox url http://testphp.vulnweb.com/listproducts.php --skip-grepping
```

**Reply by ChillVibesMushroom** [reply]
so you're saying that they found a value that matches the google api pattern @hahwul I'm currently scanning a site for work
